package.json

```diff
@@ -6,10 +6,10 @@
         "build": "nx build",
         "test": "nx test",
         "lint": "nx lint",
-        "start:all": "nx run-many --target=serve --all",
-        "build:all": "nx run-many --target=build --all",
-        "lint:all": "nx run-many --target=lint --all",
-        "test:all": "nx run-many --target=test --all --coverage",
+        "start:all": "nx run-many --target=serve --projects=api,client,dashboard,contracts --parallel=4",
+        "build:all": "nx run-many --target=build --projects=api,client,dashboard,contracts",
+        "lint:all": "nx run-many --target=lint --projects=api,client,dashboard,contracts",
+        "test:all": "nx run-many --target=test --projects=api,client,dashboard,contracts --coverage",
```

Comment:
```suggestion
        "start:all": "nx run-many --target=serve --projects=* --parallel=4",
        "build:all": "nx run-many --target=build --projects=*",
        "lint:all": "nx run-many --target=lint --projects=*",
        "test:all": "nx run-many --target=test --projects=* --coverage",
```

Reply:
This functionality is not available in the current version of nx we are using. I tried upgrading, but then there were many other dependencies breaking and the `serve` command was failing for some projects.
Reverted to `--all` for now. We can address this in a later PR if needed.